src/generated/resources/data/forge/tags/blocks/ores/dense.json

```diff
@@ -0,0 +1,9 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:coal_ore",
```

Comment:
Why is coal ore marked as dense? It drops a single item just like diamond

Reply:
Jup indeed 🤦‍♂️, thank you